content/docs/overview/osm_resource_limits.md

```diff
@@ -2,7 +2,6 @@
 title: "OSM Resource Requests and Limits"
 description: "Resource requests and limits for OSM pods and deployments"
 type: docs
-weight: 5
 ---
 
```

Comment:
Should this page have a title? Right now it is just the table

Reply:
I had no context for the purpose of the page, hence i left it as is.